about.html

```diff
@@ -153,6 +153,27 @@ <h2>2021</h2>
         </div>
       </div>
     </section>
+     <div class="about-partner-section">
+      <div class="about-partner-container">
+        <h2>Partner</h2>
+        <hr class="solid">
+        <div class="about-partner-logos">
+          <img class="heineken-logo" src="images/heineken.png" alt="Heineken">
+          <img class="cocacola-logo" src="images/coca-cola.png" alt="CocaCola">
+          <img class="samsung-logo" src="images/samsung.png" alt="Samsung">
+          <img class="spotify-logo" src="images/spotify.png" alt="Spotify">
+        </div>
+      </div>
+    </div>
+    
+    <!-- Footer Section -->
+    <footer class="about-footer-section">
```

Comment:
Please change the background color of your footer in your about to white so that your app meets the requirements 🤓

Reply:
Hey @mateo951, 
Thanks for the awesome review. Best I've ever had yet. 😊  Just to clarify, the only changes required are to change the GitHub pages to the working branch and change the background color of the footer of the about in mobile to white, right?

Reply:
Thanks alot! Glad to know you got approved 🙌